src/components/filter/filter-amount/FilterAmount.tsx

```diff
@@ -8,8 +8,8 @@ import {
 
 import {
   type FilterAmountAction,
-  type FilterAmountState,
   filterAmountOptions,
+  type FilterAmountState,
   initialFilterAmountState,
 } from 'src/components/filter/filter-amount/filterAmountReducer';
```

Comment:
I think we discussed this before, just want to make sure this is what we prefer now. `eslint-plugin-simple-import-sort` version > 9 would have a breaking change on the way they organizae the import order, which means `type` and `non-type` will be alphabetically sorted regardless it's type or not.

Reply:
I think it is fine to use the new default behavior, I don't see it as very important for the type imports to be separate. I'd rather have less custom config.